distribution/docs/src/main/resources/content/_configuring/search-options.adoc

```diff
@@ -0,0 +1,22 @@
+:title: Configuring Search Options for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 03
+:summary: Configuring local, historical, and archived Search Options for ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} provides a few options to control what metacards may be searched.
+By default, the user can perform searches that produce historical metacards, archived metacards,
+and metacards from the local catalog. If the administrator wants to, however, they can disable searching
```

Comment:
✏️ Maybe something like this would flow better?
```However, administrators have the ability to disable searching for any of these types of metacards.```

Reply:
or even `administrators can disable...` ?